_sass/_components/layout.scss

```diff
@@ -37,6 +37,7 @@
 .background-dark {
   background-color: $color-dark;
   color: $color-white;
+  -webkit-font-smoothing: antialiased;
```

Comment:
It probably would make sense to move this to the text elements below, since this will just apply the rule to the background itself, not the text elements inside it.